section-05-production-model-package/requirements/test_requirements.txt

```diff
@@ -0,0 +1,12 @@
+-r requirements.txt
+
+# testing requirements
+pytest>=6.2.3,<6.3.0
+
+# repo maintenance tooling
+black==20.8b1
+flake8>=3.9.0,<3.10.0
+mypy==0.812
+
+# TODO: check version
```

Comment:
do we have to use the kaggle thing? I would avoid it at all costs.
In real life, noone would train a model from a kaggle dataset, so I think it adds an unnecessary complication layer